t/math/declare.xml

```diff
@@ -107,16 +107,10 @@
                   <XMTok font="italic" role="UNKNOWN">x</XMTok>
                 </XMApp>
                 <XMTok role="PUNCT">,</XMTok>
-                <XMDual role="ID" xml:id="S1.Ex1.m1.10">
-                  <XMApp>
-                    <XMTok decl_id="S1.XMD3"/>
-                    <XMRef idref="S1.Ex1.m1.3"/>
-                  </XMApp>
-                  <XMApp>
-                    <XMTok name="widehat" role="OVERACCENT">^</XMTok>
-                    <XMTok decl_id="S1.XMD4" font="italic" role="ID" xml:id="S1.Ex1.m1.3">x</XMTok>
-                  </XMApp>
-                </XMDual>
+                <XMApp role="ID" xml:id="S1.Ex1.m1.10">
+                  <XMTok decl_id="S1.XMD3" name="widehat" role="OVERACCENT">^</XMTok>
+                  <XMTok decl_id="S1.XMD4" font="italic" role="ID" xml:id="S1.Ex1.m1.3">x</XMTok>
+                </XMApp>
```

Comment:
@brucemiller I rebased to the latest master this morning, and think we should consider merging this dual-cleanup PR. It has been included in the a11y showcase for a week now, all seemed in order, and the declare.xml diff is extra encouraging.

Or let me know what else you'd like me to try and improve, I've moved on from working on this one.